crates/router/src/connector/adyen/transformers.rs

```diff
@@ -182,9 +184,9 @@ impl ForeignFrom<(bool, AdyenStatus)> for storage_enums::AttemptStatus {
                 false => Self::Charged,
             },
             AdyenStatus::Cancelled => Self::Voided,
-            AdyenStatus::ChallengeShopper | AdyenStatus::RedirectShopper => {
-                Self::AuthenticationPending
-            }
+            AdyenStatus::ChallengeShopper
+            | AdyenStatus::RedirectShopper
+            | AdyenStatus::PresentToShopper => Self::AuthenticationPending,
```

Comment:
PresentToShopper is not an authentication flow. It should be mapped to Pending only

Reply:
Done👍